### PR TITLE
Support decimal label rate

### PR DESCRIPTION
**Pull request by freewym**
Make label_rate be of type float in Hubert pretraining to support decimal label rate (e.g. 33.3Hz, otherwise verify_label_lengths() will give warnings if the undelying label rate is 33.3Hz)

# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
- [x] Did you read the [contributor guideline](https://github.com/pytorch/fairseq/blob/main/CONTRIBUTING.md)?
- [ ] Did you make sure to update the docs?
- [x] Did you write any new necessary tests?

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by freewym** [comment]
@dianaml0 Any chances to merge this?

**Comment by dianaml0** [comment]
@freewym I'm just rerunning the checks and if everything passes I'll merge! Any chance you could update the PR description?

**Comment by freewym** [comment]
Thanks for your reply. I think my current description is shown as "make label_rate be of type float in Hubert pretraining to support decimal label rate (e.g. 33.3Hz, otherwise verify_label_lengths() will give warnings if the undelying label rate is 33.3Hz)"

What do you want me to change?

**Comment by dianaml0** [comment]
> Thanks for your reply. I think my current description is shown as "make label_rate be of type float in Hubert pretraining to support decimal label rate (e.g. 33.3Hz, otherwise verify_label_lengths() will give warnings if the undelying label rate is 33.3Hz)"
> 
> What do you want me to change?

@freewym no worries, I updated the title and description

**Comment by freewym** [comment]
@dianaml0 Do we need to run another round of test? I notice that the lint error has been fixed in main.

**Comment by dianaml0** [comment]
> @dianaml0 Do we need to run another round of test? I notice that the lint error has been fixed in main.

@freewym yes thanks, just updated the branch so tests should rerun

**Comment by freewym** [comment]
@dianaml0 Failed on python 3.8... Probably just a bad luck and needs a re-run

**Comment by dianaml0** [comment]
@freewym just triggered a re-run

**Comment by freewym** [comment]
@dianaml0  The internal tests failed. Not sure if it's related.

**Comment by dianaml0** [comment]
@freewym I'm rerunning the internal tests now as well

**Comment by freewym** [comment]
> @freewym I'm rerunning the internal tests now as well

@dianaml0 how was the tests going?
